nilearn/_utils/param_validation.py

```diff
@@ -63,3 +68,75 @@ def check_threshold(threshold, data, percentile_func, name='threshold'):
         raise TypeError('%s should be either a number '
                         'or a string finishing with a percent sign' % (name, ))
     return threshold
+
+
+def _get_mask_volume(mask_img):
+    """Computes the volume of a brain mask in mm^3
+    Parameters
+    ----------
+    mask_img : nibabel image object
+        Input image whose voxel dimensions are to be computed.
+
+    Returns
+    -------
+    vol : float
+        The computed volume.
+    """
+    vox_dims = mask_img.get_header().get_zooms()[:3]
```

Comment:
You need to use get_header from nilearn._utils.compat here, to avoid problems with multiple versions of nibabel.